BUILDING.md

```diff
@@ -144,18 +224,40 @@ ctest
 This will build and install binary artifacts and Python sources, create
```

Comment:
[fix] I can't add a comment to the right line, but how about changing `ctest` to `ctest --test-dir build` then you don't need to change directory (given we run all the other commands from the root of the repo).

Reply:
Yeah great idea, love the consistency. [5a4e6b2](https://github.com/OpenAssetIO/OpenAssetIO/pull/711/commits/5a4e6b2a4a782217b76cf5ec2890827a026ef7d1)